docs-guides/source/stateful-models.md

```diff
@@ -227,6 +227,217 @@ potential runtime performance improvements.
 For instance, please check out the [2024 WWDC session](https://developer.apple.com/videos/play/wwdc2024/10159/) for an 
 example that uses the [Mistral 7B model](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2)
 and utilizes the stateful prediction feature for improved performance on a GPU on a macbook pro.  
+The code for converting and deploying Mistral 7B model is released at https://huggingface.co/blog/mistral-coreml.
 
+## Example: Toy Attention Model with Stateful KV-Cache
```

Comment:
Could we put this into a test file, then simply point to it here? That way, we ensure CI would ensure the example always up-to-date.

Reply:
Yes it's already in the unit test (will be synced in next release). Here it's split from that unit test to make it more reader-friendly.